### DISCUSSION TITLE
Ability to go back to "Restart Test" after clicking tab more than once

### DISCUSSION THREAD

**Discussion by ObiKeahloa** [monkeytypegame/monkeytype#1127] [Ideas]
It would be better if there was a feature were you could go back to the "Restart Test" option after having clicked Tab more than once , maybe by using a key combination.
Like Shift+Tab

**Comment by Miodec** [comment]
Im not sure what you mean

**Comment by unrealapex** [comment]
I am confused on your question, explain a bit more.
